.github/workflows/test_itensortdvp.yml

```diff
@@ -0,0 +1,47 @@
+name: Run ITensorTDVP tests (Unbuntu)
+
+on:
+  push:
+    branches:
+      - main
+    tags: '*'
+  pull_request:
+
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ matrix.threads }} thread(s)
+    runs-on: ${{ matrix.os }}
+    env:
+      JULIA_NUM_THREADS: ${{ matrix.threads }}
+    strategy:
+      matrix:
+        version:
+          - '1.6'
+          - '1'
+        os:
+          - ubuntu-latest
+        threads:
+          - '2'
+        arch:
+          - x64
+        exclude:
+          # MacOS not available on x86
+          - {os: 'macOS-latest', arch: 'x86'}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: ${{ matrix.version }}
+          arch: ${{ matrix.arch }}
+      - name: Install Julia dependencies and run tests
+        shell: julia {0}
+        run: |
+          using Pkg;
+          Pkg.activate(temp=true);
+          Pkg.develop(path="./NDTensors");
+          Pkg.develop(path=".");
+          Pkg.add("ITensorTDVP");
+          Pkg.test("ITensorTDVP");
```

Comment:
We should also wrap this part in a `try ... catch` statement to catch the case where we knowingly break `ITensorTDVP` through a breaking release, in which case we would bump the minor version of `ITensors.jl` and Julia's package manager wouldn't allow us to add `ITensorTDVP`. See https://github.com/JuliaDiff/ChainRulesCore.jl/blob/b5df374885dbcaacda0f22175064eaa0319cee00/.github/workflows/IntegrationTest.yml#L42-L54.

Reply:
Very interesting. Please check that I got the idea right.